airbyte-webapp-e2e-tests/cypress/commands/connectorBuilder.ts

```diff
@@ -66,4 +91,54 @@ export const assertMultiPageReadItems = () => {
 
   goToTestPage(3);
   testPanelContains('[]');
-};
\ No newline at end of file
+};
+
+const MAX_NUMBER_OF_PAGES = 5;
+export const assertMaxNumberOfPages = () => {
+  assertHasNumberOfPages(MAX_NUMBER_OF_PAGES)
+};
+
+export const assertHasNumberOfSlices = (numberOfSlices: number) => {
+  cy.get('[data-testid="tag-select-slice"] button').click();
```

Comment:
Could we move the bits relying on specific selectors down into the page to keep the current separation between spec, commands and page?

E.g. 
```
cy.get('[data-testid="tag-select-slice"] button').click();
```

would become `openSlicesDropdown()` implemented in the page

and `cy.get('[data-testid="tag-select-slice"] li')` would become `getSliceDropdownOptions` and so on